### PR TITLE
Edit pass for Update mac-support-install.md. DO NOT MERGE.

### PR DESCRIPTION
**Pull request by dbyrdaquent**
Light copy edit

### CODE REVIEW

**Comment by illfated** [comment]
(Just a technical comment on one of the features of Pull Requests: it is possible to open a PR as a Draft PR instead of a standard PR. This feature is accessible from the [Create Pull Request] button. Draft PRs cannot be merged. If the priority to make sure that the PR does not get merged is a high priority, I really recommend learning how to use this feature.)

**Comment by tiburd** [comment]
Hi, Dan/@Dansimp, 

We’ve finished the edit pass of this article. Please review it and let me know if we’ve changed the technical meaning anywhere. You have five business days for the review (If you need longer, let me know). 

When your review is complete, and you’re satisfied with the changes, our team will do the final merge to confirm there are no remaining PR issues.

Thanks!

File: windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md [hunk 1]
```diff
@@ -39,13 +39,13 @@ correlation id=CB509765-70FC-4679-866D-8A14AD3F13CC
 preinstall com.microsoft.wdav end [2020-03-11 13:08:49 -0700] 804 => 1
 ```
 
-In the example above the actual reason is prefixed with `[ERROR]`.
+In the this example, the actual reason is prefixed with `[ERROR]`.
```
Comment (maximvelichko) [inline windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md:42]: "In _this_ example,"

Comment (dbyrdaquent) [reply windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md]: Fixed and committed, thank you @maximvelichko .

File: windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md [hunk 2]
```diff
@@ -39,13 +39,13 @@ correlation id=CB509765-70FC-4679-866D-8A14AD3F13CC
...
 The installation failed because a downgrade between these versions is not supported.
 
-## No MDATP's install log
+## MDATP install log fails to record installation
```
[the remark this answers is not in the document]
Comment (dbyrdaquent) [reply windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md:45]: Thank you for your comment, but I'm not sure how to change the text accurately. Do you have a suggestion for this correction? Thank you in advance.

[the remark this answers is not in the document]
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md:45]: Possible phrasing, but is this any closer to the explanation? Remember, the explanation should be in the text below, so the title should only point to what this specific error issue is about.
```suggestion
## MDATP install log empty or missing
```

[the remark this answers is not in the document]
Comment (maximvelichko) [reply windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md:45]: @illfated, I like your suggested change.
Or "MDATP install log is missing or not updated". (Every installation appends to the log. So, there can be log from previous upgrades, but not the last one.)

[the remark this answers is not in the document]
Comment (dbyrdaquent) [reply windows/security/threat-protection/microsoft-defender-atp/mac-support-install.md:45]: Changed to "MDATP install log missing or not updated". Thanks for the help!

**Pull request merged**